src/foam/u2/dialog/Popup.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+foam.CLASS({
+  package: 'foam.u2.dialog',
+  name: 'Popup',
+  extends: 'foam.u2.Element',
+
+  imports: [
+    'document'
+  ],
+  exports: [
+    'as overlay'
```

Comment:
It's ported from FOAM1; it was useful for the dialog's contents to be able to close it, eg. because they had an OK button or whatever.

I can remove it until we have a concrete use for it, if you like.